common-content/en/module/logic/search/index.md

```diff
@@ -0,0 +1,119 @@
+
+{{<tabs name="decimation">}}
+===[[Write down your solution]]===
+
+In your notebook, write down your solution to this problem. Here are the constraints again:
+
+🍾 You have **1000** wine bottles, one of which is poisoned.  
+🐀 You have **10** rats you can use to test the wine.  
+⌛ The poison is lethal and will kill within **1** hour.
+
+===[[Look at a solution]]===
+
+With just 10 rats, we need a way to make our problem simpler. Think of it as a search pattern, narrowing down our suspects with each round of testing.
+
+**Round 1:** Let's divide our 1000 bottles into 10 groups of 100. Each rat tests one group. An hour later, with one dead rat, we have narrowed our search to 100 bottles. We've saved 900 bottles of wine and we still have 9 rats to help us.
+
+**Round 2** Let's do it again: take our suspicious 100 bottles and create ten groups of ten. Our doomed rat perishes again, and after another hour, we've found the right group of ten. We now have 8 rats.
+
+**Round 3** Now we're down to just 10 bottles. Eight rats test a bottle each, leaving two untested. If a rat dies, we've found the poisoned bottle. If not, we have two bottles left for the last round.
+
+**Round 4: Base case** We can use just one rat to test one bottle. If we have a dead rat, it's the poisoned bottle. If not, the last untasted bottle is the culprit.
+
+Through careful division and patience, we've turned an overwhelming search into a manageable investigation. Four hours, and we'll have our answer.
+
+{{</tabs>}}
+
+> #### New condition
+>
+> The birthday party is now in **1** hour, so you only have time for one test!
+
+{{<tabs name="binary">}}
+===[[Write down your solution]]===
+Think about how you can use some of the thinking tools you just learned to solve this problem.
+
+You will need boolean logic, deduction, and binary at least.
+
+===[[Look at a solution]]===
+
+If we only have one hour, we can only do a single round of testing! Our strategy of narrowing down can't work. We're going to have to think about what we have learned about binary to solve this problem.
+
+I really advise that you write this out yourself in binary as you follow the solution through.
+
+Let's reduce our case. Let's say we have 3 bottles and 2 rats.
+
+We're going to number the bottles in binary. Each rat will test a different _bit_ of the binary number. Rat 1 will test the 1s place, Rat 2 the 2s place.
+
+| Decimal | Binary | Rat 1 | Rat 2 |
+| ------- | ------ | ----- | ----- |
+| 1       | 001    | x     |       |
+| 2       | 010    |       | x     |
+| 3       | 011    | x     | x     |
+
+If rat 1 dies, we know the poison is in the 1s place. It's bottle 1. If rat 2 dies, it's bottle 2. If both die, it's bottle 3.
+
+To be clear: instead of testing bottles one by one, we will have the rats sip from _multiple_ bottles at once. Each rat's survival or death becomes a piece of information that we can _combine_ to locate the poisoned bottle.
```

Comment:
```suggestion
To be clear: Before we were linking rats to separate bottles, and only one rat told us information about any one bottle. And at most one rat would die per round. Instead, we will have the rats sip from _multiple_ bottles at once. Each rat's survival or death becomes a piece of information that we can _combine_ with information from other rates to locate the poisoned bottle.
```

Reply:
No they are still doing it in groups in the decimation round. I will make that clearer up there.